### PR TITLE
Feat(eos_designs): Allow 'evpn_vlan_bundle' to be set up at tenant level

### LINKED ISSUE #3408: Allow evpn_vlan_bundle to be set up at tenant level
### Enhancement summary

Allow `evpn_vlan_bundle` variable to be set up at tenant level, so all VLANs defined can be assigned to the same VLAN bundle.

### Which component of AVD is impacted

eos_designs

### Use case example

```
tenants:
  - name: TENANT_A
     vrfs:
       PRODUCTION:
       (...)
     l2vlans:
     - id: 1
       name: "TEST1"
       evpn_vlan_bundle: "VLAN_BUNLE_1"
     - id: 2
       name: "TEST2"
       evpn_vlan_bundle: "VLAN_BUNLE_1"
```

### Describe the solution you would like

```
tenants:
  - name: TENANT_A
    evpn_vlan_bundle: "VLAN_BUNLE_1"
     vrfs:
       PRODUCTION:
       (...)
     l2vlans:
     - id: 1
       name: "TEST1"
     - id: 2
       name: "TEST2"
```

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by laxmikantchintakindi**
## Change Summary

Allow evpn_vlan_bundle variable to be set up at tenant level, so all VLANs defined can be assigned to the same VLAN bundle.

## Related Issue(s)

Fixes #3408

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes

Allow evpn_vlan_bundle variable to be set up at tenant level, so all VLANs defined can be assigned to the same VLAN bundle.
```
tenants:
  - name: TENANT_A
    evpn_vlan_bundle: "VLAN_BUNLE_1"
     vrfs:
       PRODUCTION:
       (...)
     l2vlans:
     - id: 1
       name: "TEST1"
     - id: 2
       name: "TEST2"
```

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/filtered_tenants.py [hunk 1]
```diff
@@ -267,6 +272,10 @@ def filtered_vrfs(self: SharedUtils, tenant: dict) -> list[dict]:
             if vrf["svis"] or vrf["l3_interfaces"] or vrf["loopbacks"] or self.is_forced_vrf(vrf):
                 filtered_vrfs.append(vrf)
 
+            if tenant_evpn_vlan_bundle := get(tenant, "evpn_vlan_bundle"):
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/filtered_tenants.py:275]: i am not sure that we should do it for SVI too under vrf. 
@xaviramon is this ok if `evpn_vlan_bundle ` set on tenant level than configure VRF's SVI under that bundle.
cc: @ClausHolbechArista

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml [hunk 2]
```diff
@@ -79,6 +79,11 @@ $defs:
             By default the iBGP peering subnet is redistributed into the overlay routing protocol per VRF.
             Setting `redistribute_mlag_ibgp_peering_vrfs: false` under a tenant will change this default to prevent redistribution of these subnets for all VRFs in the tenant.
             This setting can be overridden per VRF.
+        evpn_vlan_bundle:
+          type: str
+          description: |-
+            Name of the "evpn_vlan_bundle" defined under "evpn_vlan_bundles".
+            This settings enables the "evpn_vlan_bundle" setting for all l2vlans and SVIs present in the tenant.
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml]: Name of a bundle defined under 'evpn_vlan_bundles' to inherit configuration for all l2vlans and SVIs present in the tenant.

File: ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/filtered_tenants.py (hunk 1 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/filtered_tenants.py:275]: It is fine to just have this on the tenant for now. Thanks

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml (hunk 2 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml]: None of them are really good IMO.

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_network_services.schema.yml]: Added better description.

**Review by ClausHolbechArista — APPROVED** [review]

**Review by MaheshGSLAB — APPROVED** [review]

**Pull request merged**